### PR TITLE
Feature/tmp fly storage unit test

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: models/classes/upload/TempFlyStorageAssociation.php [hunk 1]
```diff
@@ -20,15 +20,19 @@
...
 
-use common_Utils;
 use oat\oatbox\AbstractRegistry;
 use oat\oatbox\filesystem\File;
+use Zend\ServiceManager\ServiceLocatorAwareInterface;
+use Zend\ServiceManager\ServiceLocatorAwareTrait;
+use Zend\ServiceManager\ServiceLocatorInterface;
 
-class TempFlyStorageAssociation extends AbstractRegistry implements TmpLocalAwareStorageInterface
+class TempFlyStorageAssociation extends AbstractRegistry implements TmpLocalAwareStorageInterface, ServiceLocatorAwareInterface
```
Comment (boajer) [inline models/classes/upload/TempFlyStorageAssociation.php]: Is it possible to get rid of `AbstractRegistry` compatibility? ( to have proper service without any static methods etc )

Comment (hutnikau) [reply models/classes/upload/TempFlyStorageAssociation.php]: Yes, i will did that in another branch. This one is needed just to create a unit test in order to be sure that i will not break anything but that refactoring.

File: test/unit/upload/TempFlyStorageAssociationTest.php [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        $pmProphecy->setServiceLocator(Argument::any())->willReturn(null);
+        $pmProphecy->getPersistenceById('default_kv')
+            ->willReturn(new \common_persistence_KeyValuePersistence([], new \common_persistence_InMemoryKvDriver()));
+        $pm = $pmProphecy->reveal();
+
+        $serviceManager->register(PersistenceManager::SERVICE_ID, $pm);
+
+        return $serviceManager;
+    }
+}
```
Comment (boajer) [inline test/unit/upload/TempFlyStorageAssociationTest.php]: Must ends with a new line according PSR-2

File: models/classes/upload/TmpLocalAwareStorageInterface.php [hunk 3]
```diff
@@ -45,7 +45,4 @@ public function getLocalCopies(File $file);
      * @param File $file
      */
     public function removeFiles(File $file);
-
-    /** @return $this */
-    public static function getStorage();
 }
```
Comment (boajer) [inline models/classes/upload/TmpLocalAwareStorageInterface.php]: Please add linebreak

File: models/classes/upload/TempFlyStorageAssociation.php [hunk 4]
```diff
@@ -92,4 +89,55 @@ public function removeFiles(File $file)
...
+    }
+
+    /**
+     * @return string
+     */
+    private function getPrefix()
+    {
+        return static::class;
+    }
 }
```
Comment (boajer) [inline models/classes/upload/TempFlyStorageAssociation.php]: Please add linebreak

**Pull request merged**